src/Service/EventBridge/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## NOT RELEASED
 
+### Added
+
+- AWS api-change: EventBridge now supports Resource-based policy authorization on event buses. This enables cross-account PutEvents API calls, creating cross-account rules, and simplifies permission management.
```

Comment:
this changelog looks related to the diff but does not reflect what has been done in async-aws...

Reply:
Lets remove it